### DISCUSSION TITLE
IMU sensors as a sole tracking method.

### DISCUSSION THREAD

**Discussion by MRezaEffatparvar** [PlusToolkit/PlusLib#1186] [Q&A] [answered]
Hello everyone,
I am currently using an OptiTrack system for freehand ultrasound. Could you recommend a much more portable motion-tracking system to replace OptiTrack? Can IMU sensors be used as the sole tracking method? If not, are there any other more affordable and more portable options you could suggest? I really like to have the freehand option that I can use on the go.
Thanks for any help

**Comment by ungi** [comment]
Hi, if line of sight or the large optical marker is the issue, you could consider electromagnetic tracking. I had good experience with Ascension (NDI) TrakSTAR.

**Reply by MRezaEffatparvar** [reply]
Thanks for your quick response. Yes, this could be a great idea. However, as the configuration files have been made for IMU sensors, I wonder if they could be used as the sole motion tracking sensor?

**Reply by ungi** [reply]
As far as I know, speckle tracking and IMU sensors can give you position tracking with 10% error, and it only works if you move the ultrasound in one general direction, doing one sweep. If that is enough for your application, then it should work in theory. But that tracking methods is currently not available in PLUS or Slicer.

**Reply by MRezaEffatparvar** [reply]
Yes, we will use them primarily for one direction. If they are not currently available, does that mean the example configuration files won't work anymore? For instance, consider Phidgets Spatial as an example

**Reply by ungi** [reply, marked as answer]
You can collect IMU data using the current configuration. But IMU data will result in correct 3D reconstruction if you only tilt the probe and don't move it on the skin surface. If you sweep on the skin surface, you will need to add speckle tracking (or another method to estimate translation). Those approximate image-based tracking methods are outside the scope of PLUS.
